src/data/ACTIONS/root/MNK.ts

```diff
@@ -37,7 +44,6 @@ export const MNK = ensureActions({
 		icon: 'https://xivapi.com/i/000000/000213.png',
 		onGcd: true,
 		speedAttribute: Attribute.SKILL_SPEED,
-		potency: 280,
```

Comment:
Let's not break the Twin Snakes module.

Reply:
Migrated/fixed the MNK potency definitions to use the new system.